lib/src/logger.dart

```diff
@@ -43,13 +43,14 @@ class Logger {
   /// All logs with levels below this level will be omitted.
   static Level level = Level.verbose;
 
+  @Deprecated("")
```

Comment:
Why did you deprecate these private variables?

Reply:
It got accidentally committed, I'll amend.

I was experimenting with the field and pondering deprecating the callback functionality. It's not used as far as I can tell (no tests, not documented) and I think it would simplify the code. Also I could not think of a scenario in which this would be handy/required. Custom printers/outputs should do. But perhaps I'm missing something?

Reply:
> Custom printers/outputs should do.

I agree :+1: